.projenrc.ts

```diff
@@ -39,7 +39,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   tsconfigDev: { compilerOptions: { ...commonTscOptions } },
   // dependency config
   jsiiVersion: '~5.0.0',
-  cdkVersion: '2.93.0',
+  cdkVersion: '2.95.1',
```

Comment:
I have no problem with bumping CDK version. I think we bump to latest if you'd like.

Reply:
Bumped to the latest version, seems fine as far as I can tell!